web/frontend/package.json

```diff
@@ -8,7 +8,7 @@
     "@mdi/js": "^6.9.96",
     "@viamrobotics/prime": "^0.0.79",
     "@viamrobotics/rpc": "^0.1.32",
-    "@viamrobotics/sdk": "^0.0.6",
+    "@viamrobotics/sdk": "file:../../../../viamrobotics/viam-typescript-sdk/viamrobotics-sdk-0.0.8.tgz",
```

Comment:
bump version once https://github.com/viamrobotics/viam-typescript-sdk/pull/6 is merged and the corresponding package is released